platform/android/java_godot_wrapper.cpp

```diff
@@ -109,6 +109,16 @@ jobject GodotJavaWrapper::get_class_loader() {
 	}
 }
 
+GodotJavaViewWrapper *GodotJavaWrapper::get_godot_view() {
+	if (_godot_view != NULL) {
+		return _godot_view;
+	}
+	JNIEnv *env = ThreadAndroid::get_env();
+	jfieldID godot_view_field_id = env->GetFieldID(godot_class, "mRenderView", "Lorg/godotengine/godot/GodotRenderView;");
```

Comment:
Add a `private` getter instead to the `Godot.java` class and retrieve the view through it. This helps reduce hard to track bugs if the view's name get changed. For example the view's name changes between the `master` and `3.2` branch while the getter on the other hand would remain the same.

In general, try to avoid direct access to a class field, both from the java and/or native side, as this tends to break encapsulation.

Reply:
Sorry, Im coming from c# where everything is a getter 😅

Reply:
No worries, it's the same in Kotlin as well, so sometimes it can get confusing :)